src/utils.js

```diff
@@ -63,6 +66,10 @@ export function correctDefault(parameter, type) {
         return {};
       }
 
+    // Don't allow defaults for the base64 parameter, this could be dangerous.
```

Comment:
That's a good point... I haven't looked at the brewtils PR yet but we should probably prevent defaults there as well?

Reply:
@hazmat345 I've inserted code into `Brewtils` to do the same, though I'm pretty sure this code snippet is what was doing the prevention for me.